pytradfri/device/base_controller.py

```diff
@@ -1,13 +1,24 @@
 """Base class for a controller."""
 from __future__ import annotations
 
-from typing import TYPE_CHECKING
+from abc import abstractmethod
+from typing import TYPE_CHECKING, Sequence
+
+from pydantic import BaseModel, Field
+
+from ..const import ATTR_ID
 
 if TYPE_CHECKING:
     # avoid cyclic import at runtime.
     from . import Device
 
 
+class BaseResponse(BaseModel):
```

Comment:
Maybe move this to the resource module since we now use it in other modules than the device package?

Reply:
Makes sense